apisix/cli/apisix.lua

```diff
@@ -21,11 +21,12 @@ local pkg_path_org = package.path
 local apisix_home = "/usr/local/apisix"
 local pkg_cpath = apisix_home .. "/deps/lib64/lua/5.1/?.so;"
                   .. apisix_home .. "/deps/lib/lua/5.1/?.so;"
-local pkg_path = apisix_home .. "/deps/share/lua/5.1/?.lua;"
+local pkg_path_deps = apisix_home .. "/deps/share/lua/5.1/?.lua;"
+local pkg_path_env = ";" .. apisix_home .. "/?.lua;"
 
 -- modify the load path to load our dependencies
 package.cpath = pkg_cpath .. pkg_cpath_org
-package.path  = pkg_path .. pkg_path_org
+package.path  = pkg_path_deps .. pkg_path_org .. pkg_path_env
```

Comment:
Based on @spacewander  suggestion, shouldn't we put `pkg_path_env` in front?

Reply:
> Based on @spacewander suggestion, shouldn't we put `pkg_path_env` in front?

no, the pkg_path_org is `'./apisix/cli/env.lua'` , and the pkg_path_env is `/usr/local/apisix/apisix/cli/env.lua`, so `pkg_path_env` should be put after `pkg_path_org`